hazelcast/src/test/java/com/hazelcast/query/impl/predicates/PredicatesTest.java

```diff
@@ -76,6 +76,8 @@
 @Category(QuickTest.class)
 public class PredicatesTest extends HazelcastTestSupport {
 
+    private static final String DUMMY = "DUMMY_ATTRIBUTE_IGNORED";
```

Comment:
a nitpicking: `DUMMY_ATTRIBUTE_NAME` would be better. or just an `ATTRIBUTE_NAME`